nixos/modules/services/networking/connman.nix

```diff
@@ -89,11 +96,13 @@ in {
       assertion = !config.networking.useDHCP;
       message = "You can not use services.connman with networking.useDHCP";
     }{
+      # TODO: connman seemingly can be used along network manager and
+      # connmanFull supports this - so this should be worked out somehow
       assertion = !config.networking.networkmanager.enable;
       message = "You can not use services.connman with networking.networkmanager";
     }];
```

Comment:
can you also open an issue? We have a tracker for these things.

Reply:
@worldofpeace it's just that I personally never have used or tried to use network manager at all. It'll take me quiet an effort to write a good bug report so I think it'd be proper to wait for someone else to want that feature. Hopefully he'll encounter this comment indicating at least someone was aware of this..